uavcan/equipment/esc/1036.StatusExtended.uavcan

```diff
@@ -0,0 +1,15 @@
+#
+# Extended Generic ESC status.
+# Additional fields on top of the generic ESC status message.
+# Transmitted at a lower rate.
+#
+
+float16 motor_temperature   # Temperature of connected motor, in Kelvin.
```

Comment:
Use degrees C so you get more resolution in realistic ranges

Reply:
I'm happy to do that, however will that cause confusion generally with other units following the Kelvin route?